### DISCUSSION TITLE
CustomText separated in subfolders

### DISCUSSION THREAD

**Discussion by C4BR3R4** [Zrips/CMI#6191] [Enhancement]
Due to the amount of CustomText we have, it would be appreciated if the files could be separated in subfolders.

**Comment by mrfloris** [comment]
In CMI 9.0.3.0 and up, currently especially in 9.0.4.2, all the CustomAlias.yml files are in a directory: 
`~/plugins/CMI/CustomAlias/`

<img width="294" alt="Screenshot 2021-09-21 at 13 03 45" src="https://user-images.githubusercontent.com/28841349/134159775-667ca697-bfb2-4385-9584-c1bec3777947.png">

You can take the commands out of CustomAlias.yml and put them in your own filenames. Just have it say customalias at the top like so: (staff.yml)
```yaml
# All my CustomAlias staff commands:
CustomAlias:
  command ...
```

I think personally it's a bit overkill to make a yml file for Each command and have them sub into groups inside a group already.

**Comment by C4BR3R4** [comment]
These are not the CustomAlias files, I'm referring to the CustomText files.

![imagen](https://user-images.githubusercontent.com/4703920/134210993-aaaba661-1cf4-4509-ba1f-cad2f0abd9f9.png)

**Comment by mrfloris** [comment]
I have no idea why I completely misread custom ALIAS vs TEXT. Embarrassing.

Yeah, they can add up pretty quickly in the customTEXT directory as well. At least they're their own file already, but wouldn't hurt to be able to make a few categories like publicTxt/ staffTxt/ eventTxt/
